packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTypeaheadSelectList/CWTypeaheadSelectList.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../../../../styles/shared';
+
+.TypeaheadSelectList {
+  width: 400px;
+
+  .SelectList {
+    font-family: $font-family-neue-haas-unica;
+    border: 1px solid $neutral-200;
+    padding: 10px 16px;
+    border-radius: $border-radius-corners-wider;
+
+    &:hover {
+      border: 1px solid $primary-600;
+    }
+
+    .tasl__value-container {
```

Comment:
@Miaplacidus @mzparacha I just want to highlight that we have possible merge conflicts in the are of SelectList because of this PR and another one https://github.com/hicommonwealth/commonwealth/pull/5086/files 

1. I just want to make sure that you guys are on the same page and that we have generic `SelectList` component that do not clash with any other styles.
2. Ideally we should have one approach to style react-select component. In this PR `classNamePrefix` is used, however in @mzparacha's PR, `[class$='-ValueContainer']` approach is used. For me, former looks a bit cleaner tbh.

Reply:
> I just want to make sure that you guys are on the same page and that we have generic SelectList component that do not clash with any other styles.

Thanks for the heads-up @masvelio.

> Ideally we should have one approach to style react-select component. In this PR classNamePrefix is used, however in @mzparacha's PR, [class$='-ValueContainer'] approach is used. For me, former looks a bit cleaner tbh.

Sure, so I think we should update 1 PR with all the necessary changes? Or merge a pr and then make the necessary adjustments in the other?

cc: @masvelio @Miaplacidus

Reply:
probably would be good to merge one and then another with resolved conflicts

Reply:
@masvelio @mzparacha
The new typeahead dropdown is built on top of `CWSelectList`. Since you've modified the component on top of which my own is built, it's probably best to get your work merged in first and then I can modify my own work accordingly.

Reply:
Sounds good! Thanks

Reply:
Update: I bumped my pr for merge [here](https://github.com/hicommonwealth/commonwealth/pull/5086#issuecomment-1746889519)